### ISSUE TITLE
SQL syntax error when running populate_xtdcomments command using Postgresql

### ISSUE THREAD

**Issue by ramonakira** [danirus/django-comments-xtd#149]
Hi,

When running `python manage.py populate_xtdcomments` I get the following error:

```
ERROR:  syntax error at or near "'comment_ptr_id'" at character 52
STATEMENT:  INSERT INTO django_comments_xtd_xtdcomment        ('comment_ptr_id', 'thread_id', 'parent_id',        'level', 'order', 'followup') VALUES (16360, 16360, 16360, 0, 1, 0)
```

When I then change the SQL to read
```
INSERT INTO django_comments_xtd_xtdcomment ("comment_ptr_id", "thread_id", "parent_id", "level", "order", "followup") VALUES (16360, 16360, 16360, 0, 1, 0)
```
(so using double quotes instead of single quotes) 

I get the following error:

```
ERROR:  column "followup" is of type boolean but expression is of type integer
LINE 1: ..., "order", "followup") VALUES (16360, 16360, 16360, 0, 1, 0)
                                                                     ^
HINT:  You will need to rewrite or cast the expression.
```

Not sure how this should be solved though. I am using Postgresql 9.6, Python 3.6 and Django 2.2.12. 

Thanks in advance!

**Comment by ramonakira** [comment]
As a followup; for Postgres the correct sql is:
```
INSERT INTO django_comments_xtd_xtdcomment        ("comment_ptr_id", "thread_id", "parent_id",        "level", "order", "followup") VALUES (16360, 16360, 16360, 0, 1, FALSE)
```
So the last `0` corresponding to `followup` should be `FALSE`

**Comment by danirus** [comment]
Thanks for the report @ramonakira!

**Comment by danirus** [comment]
Also thanks for the quick fix. Your suggestion is valid for all the officially Django supported database engines: switching the literal value in the INSERT SQL statement from 0 to FALSE.

As of this writing a literal `FALSE` value is accepted by [MariaDB](https://mariadb.com/kb/en/sql-language-structure-boolean-literals/), [PostgreSQL](https://www.postgresql.org/docs/9.1/datatype-boolean.html), [MySQL](https://dev.mysql.com/doc/refman/8.0/en/boolean-literals.html), [Oracle](https://docs.oracle.com/cd/B12037_01/appdev.101/b10807/img_text/boolean_literal.htm) and [SQLite](https://www.sqlite.org/quirks.html) (see section 3.1: literals TRUE and FALSE accepted as of v3.23.0).

**Comment by danirus** [comment]
Fixed in master.
Will be released in v2.6.2.

**Comment by danirus** [comment]
Thanks again for the report. 
The issue has been resolved and fixed in the pull request #177.
